staNMF/staNMF.py

```diff
@@ -371,10 +392,16 @@ def instability(self, k1=0, k2=0):
                 self.instabilitydict[k] = (np.sum(distMat) / (numReplicates *
                                            (numReplicates-1)))
 
-                outputfile = open("instability.csv", "w")
-                for i in sorted(self.instabilitydict):
-                    outputfile.write(str(i) + "," +
-                                     str(self.instabilitydict[i]) + "\n")
+                if self.parallel:
```

Comment:
If you set parallel, it saves each K's instability calculation in its respective K folder. In this case, the plot function has to open a separate file for each K's instability. This allows it to write multiple instability calculations without overwriting anything. If parallel=False, it saves them all together, in the next folder out, and in turn can read them all in at once. 

I thought it might be less energetically expensive to just write this single line as a formatted string, so I don't have to initialize a csvwriter(). Do you think it'd be better that way?